pkg/legacy/converter.go

```diff
@@ -47,7 +47,10 @@ func FromAgentConfig(agentConfig Config) error {
 		config.Datadog.Set("forwarder_timeout", value)
 	}
 
-	// TODO: default_integration_http_timeout
+	if value, err := strconv.Atoi(agentConfig["default_integration_http_timeout"]); err == nil {
+		config.Datadog.Set("default_integration_http_timeout", value)
```

Comment:
We should log an error if we were not able to parse `default_integration_http_timeout`.

Reply:
Should I do that for all of these?

Reply:
Good question ... We may flood the user a bit. Should we stay silent ?

Reply:
I think that may be best, though it's up to you.

Reply:
👍